lib/matplotlib/backend_tools.py

```diff
@@ -358,101 +356,66 @@ def remove_rubberband(self):
         pass
 
 
-class ToolQuit(ToolBase):
+class _ToolForwardingToClassicToolbar(ToolBase):
+    _rc_entry = ''  # Must be set by subclass.
+    default_keymap = property(lambda self: mpl.rcParams[self._rc_entry])
+
+    def trigger(self, sender, event, data=None):
+        sentinel = str(uuid.uuid4())
+        # Trigger class toolbar implementation by temporarily setting the keymap to a
+        # unique key and sending an appropriate event.
+        with cbook._setattr_cm(event, key=sentinel), \
```

Comment:
Overall I don't understand what this is doing (which is likely due to my lack of understanding of the event mechanism, but OTOH that likely applies to more contributors):

Why do we need the sentinel? / How does the triggering work? AFAICS, most of `key_press_handler()` is `if event.key =  ...` blocks, which don't get triggered by the sentinel.

Reply:
This is actually modified from the comment you originally suggested in https://github.com/matplotlib/matplotlib/pull/16823#discussion_r394652249 :-)  Does that ring a bell to you, and/or do you think this can be made clearer salvageable with minor edits?  Otherwise I can try to rewrite something more detailed from scratch.

Reply:
Ok, understand now again  what we are doing.  it's just the wording that confused me. Do the proposed changes make sense? AFAICS, the key press event handler is used by the classic Toolbar, but it's a more fundamental concept, and we're not directly relying on the classic Toolbar here.